src/plugins/workspace/opensearch_dashboards.json

```diff
@@ -5,8 +5,9 @@
   "ui": true,
   "requiredPlugins": [
     "savedObjects",
-    "opensearchDashboardsReact"
+    "opensearchDashboardsReact",
+    "management"
```

Comment:
any reason move it to requiredPlugins?

Reply:
There is a reference in the code: `There is a reference in the code.`

Reply:
Seems no need to include management as required bundles.

Reply:
I think adding it as require_bundles is good enough.

Reply:
If management is not in required bundles, CI will failed.

Reply:
Forget it, it seems requiredBundles has some impact on bundling lifeCycle. Then maybe let's revert to hard-coded version then, sorry for the back-and-forth.

Reply:
It's not a matter, the code has been reverted.